tests/integration/tests/test_gateway.py

```diff
@@ -11,6 +11,31 @@
 LOG = logging.getLogger(__name__)
 
 
+def check_gateway_service_and_port(p):
+    gateway_http_port = None
+    services = json.loads(p.stdout.decode())
+
+    gateway_services = [
+        svc
+        for svc in services["items"]
+        if (
+            svc["metadata"].get("labels").get("projectcontour.io/owning-gateway-name")
+            == "my-gateway"
+            or svc["metadata"].get("labels").get("io.cilium.gateway/owning-gateway")
```

Comment:
meh, do contour and cilium really have different labels `owing-gateway` and `owning-gateway-name`? That sucks.